src/views/studio/MyVideosView/EmptyVideosView.tsx

```diff
@@ -87,38 +92,39 @@ const MessageView = styled.div`
 `
 
 // for tabs
-export const EmptyVideos = () => {
+export const EmptyVideos: React.FC<{ isDraftTab: boolean }> = ({ isDraftTab }) => {
   return (
     <Container>
-      <WellIllustration />
+      <TheaterMaskIllustration />
       <Message>
-        <Text variant="h5">No videos found...</Text>
+        <Text secondary variant="body2">
+          There are no {isDraftTab ? 'drafts' : 'videos'} currently
```

Comment:
In designs texts are specified to each tab, like `Currently there are no unlisted videos` for unlisted etc.
Maybe we should do that here as well